dutyfree-chart/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: dutyfree
+description: A Helm chart for deploying dutyfree (concourse resources website) on kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application.
+appVersion: latest
```

Comment:
got me thinking the same, but seems like anything can go on there:

> Note that the appVersion field is not related to the version field. It is a way
> of specifying the version of the application. For example, the drupal chart may
> have an appVersion: 8.2.1, indicating that the version of Drupal included in the
> chart (by default) is 8.2.1. **This field is informational, and has no impact on
> chart version calculations.**

which, makes sense - it could be that the underlying software is not semver'ed, having some weird versioning scheme 🤔

Reply:
we still don't have any versioning scheme. that's right